Tests/UnitTests/TestHelpers/TestLogHandler.swift

```diff
@@ -75,6 +75,12 @@ final class TestLogHandler {
 
     deinit { Self.sharedHandler.remove(observer: self) }
 
+    /// If a test overrides `Purchases.verboseLogHandler` or `Logger.internalLogHandler`
+    /// this needs to be called to re-install the test handler.
```

Comment:
Is it worth doing this before every test? Just so we don't forget in the future

Reply:
That would break with `TestLogHandler` because it gets installed only once.